### ISSUE TITLE
cluster MOVE redirect

### ISSUE THREAD

**Issue by knavely** [debasishg/scala-redis#115]
when running unstable branch redis in cluster mode configured locally with 6 nodes, 3 masters and 3 slaves as described here: http://redis.io/topics/cluster-tutorial

replication steps: 
1. as in the example create the redis cluster r with 3 nodes: localhost:7000, localhost:7001, localhost:7002.  
2. r.lpush(1,"hi") -- worked
3. r.lpush(4,"hi") should redirect to localhost:7002 where redis returns a move redirect, not a long, i suspect from looking at the stack trace that the client expect a long.

see the section on MOVE redirect: http://redis.io/topics/cluster-spec

**Comment by chitralverma** [comment]
Is this issue still open ?

**Comment by debasishg** [comment]
Yeah .. it's still open .. unfortunately I don't have any time to implement redis cluster spec. PRs welcome though :-)
